lib/server.js

```diff
@@ -0,0 +1,18 @@
+const killContainers = require('../lib/util').killContainers;
+const cron = require('node-cron');
+const config = require('config');
+const express = require('express');
+const app = express();
+
+const name = config.get('name');
+const age = config.get('age');
+const range = config.get('range');
+
+const PORT = 3000;
+
+app.listen(PORT, err => {
+    cron.schedule(range, async () => {
+        console.log(`Deleting containers with name: ${name} and age: ${age}`);
+        await killContainers({name, age});
+    });
+});
```

Comment:
express is not needed here since we're not making a REST api here (unless was planning to change the design of the tool)
```suggestion
cron.schedule(range, async () => {
    console.log(`Deleting containers with name: ${name} and age: ${age}`);
    await killContainers({name, age});
});
```

If removing this, you can also remove express import on line 4-5 as well.